src/flutter/shell/platform/linux_embedded/surface/context_egl.h

```diff
@@ -6,6 +6,7 @@
 #define FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_CONTEXT_EGL_H_
```

Comment:
You shouldn't write a specific code for EGLStream in `context_egl_eglsream.h`.

Reply:
I moved a specific code for EGLstream to `context_egl_drm_eglstream.h`.